### PR TITLE
Bugfix/523 unable to import search project

### PR DESCRIPTION
**Pull request by MdeBoer95**

### CODE REVIEW

File: inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/exporter/ExportedKnowledgeBase.java [hunk 1]
```diff
@@ -208,7 +242,27 @@ public void setBasePrefix(String aBasePrefix)
...
-    
+
+    public List<String> getExplicitlyDefinedRootConcepts()
+    {
+        return explicitlyDefinedRootConcepts;
+    }
+
+    public void setExplicitlyDefinedRootConcepts(List<String> explicitlyDefinedRootConcepts)
+    {
+        this.explicitlyDefinedRootConcepts = explicitlyDefinedRootConcepts;
```
Comment (reckart) [inline inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/exporter/ExportedKnowledgeBase.java]: `explicitlyDefinedRootConcepts = aExplicitlyDefinedRootConcepts;`

File: inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/exporter/ExportedKnowledgeBase.java [hunk 2]
```diff
@@ -208,7 +242,27 @@ public void setBasePrefix(String aBasePrefix)
...
+    }
+
+    public String getDefaultLanguage()
+    {
+        return defaultLanguage;
+    }
+
+    public void setDefaultLanguage(String defaultLanguage)
+    {
+        this.defaultLanguage = defaultLanguage;
```
Comment (reckart) [inline inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/exporter/ExportedKnowledgeBase.java]: `defaultLanguage = aDefaultLanguage;`

File: inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/exporter/ExportedKnowledgeBase.java [hunk 3]
```diff
@@ -60,13 +68,19 @@
...
+
     @JsonProperty("support_concept_linking")
     private boolean supportConceptLinking;
-    
+
     @JsonProperty("base_prefix")
     private String basePrefix;
-    
+
+    @JsonProperty("explicitly-defined-root-concepts")
```
Comment (reckart) [inline inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/exporter/ExportedKnowledgeBase.java]: Could we please simply call this `root-concepts`?

File: inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/exporter/ExportedKnowledgeBase.java [hunk 4]
```diff
@@ -52,6 +54,12 @@
     @JsonProperty("property_type_iri")
     private String propertyTypeIri;
 
+    @JsonProperty("property-label-iri")
```
Comment (reckart) [inline inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/exporter/ExportedKnowledgeBase.java]: We seem to be mixing naming styles here, i.e. `property_type` vs. `property-label`. One style only please!

**Comment by naveen2507** [comment]
Works

**Pull request merged**